### PR TITLE
blog: This month in Servo (September 2024)

### PR DESCRIPTION
**Pull request by delan**

### CODE REVIEW

**Comment by sagudev** [comment]
Beware that https://github.com/servo/servo/pull/33504 was "reverted". See https://github.com/servo/servo/pull/33504#issuecomment-2385442125 so you can just omit it from blog.

File: _posts/2024-09-30-this-month-in-servo.md [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+
+Our table layout has improved, with support for **width** and **height presentational attributes** (@Loirooriol, #33405, #33425), as well as better handling of **‘border-collapse’** (@Loirooriol, #33452) and **extra &lt;col> and &lt;colgroup> columns** (@Loirooriol, #33451).
+
+We’ve also started working on the intrinsic sizing keywords **‘min-content’**, **‘max-content’**, **‘fit-content’**, and **‘stretch’** (@Loirooriol, #33492).
+Before we can support them, though, we needed to land patches to calculate intrinsic sizes, including for **percent units** (@Loirooriol, #33204), **aspect ratios** of replaced elements (@Loirooriol, #33240), **column flex containers** (@Loirooriol, #33299), and **‘white-space’** (@Loirooriol, #33343).
+
+We’ve heavily reworked our **WebGPU support**, with **faster canvas presentation** (@sagudev, #33387), support for **resizing the canvas** (@sagudev, #33521) and the **‘constants’ property on GPUProgrammableStage** (@sagudev, #33291), plus improvements to **GPUBuffer** (@sagudev, #33154) and the **destroy() method on GPUTexture** (@sagudev, #33534).
```
Comment (delan) [inline _posts/2024-09-30-this-month-in-servo.md:384]: @sagudev, is this accurate? in particular, is it accurate to say that the web-facing change in #33521 (after splitting #30479 out from #30386) is that contexts can now handle the canvas being resized? or was it also not possible to unconfigure() and reconfigure() before, or something else?

Comment (sagudev) [reply _posts/2024-09-30-this-month-in-servo.md:384]: https://github.com/servo/servo/pull/33521 was a lot, in addition to implementing resize for canvas it also completely overhauled GPUCanvasContext to match the spec (getcurrenttexture didn't work properly before and reconfigure also didn't work properly, hence so many new good expectations). I would describe it as "reworked webgpu presentation to match the spec and properly support (re)configuration and resize".

Comment (sagudev) [inline _posts/2024-09-30-this-month-in-servo.md:384]:
```suggestion
We’ve also worked on our **WebGPU support**, with **faster canvas presentation** (@sagudev, #33387), **support pipeline-overridable constants** (@sagudev, #33291), completely reworked **GPUBuffer** to match the spec (@sagudev, #33154) and heavily reworked webgpu presentation to match the spec and properly support (re)configuration and resize on **GPUCanvasContext** (@sagudev, #33521).
```

Comment (delan) [reply _posts/2024-09-30-this-month-in-servo.md:387]: thanks! done in d4ed0826d05fd665103d956ac11add4ae5d487c6

Comment (delan) [reply _posts/2024-09-30-this-month-in-servo.md:387]: thanks, i appreciate the info :)

**Pull request merged**